protected/views/patient/_patient_all_episodes.php

```diff
@@ -32,7 +32,7 @@
 			<tr>
 				<th>Start Date</th>
 				<th>End Date</th>
-				<th>Firm</th>
+				<th>Context</th>
```

Comment:
You shouldn't use the literal "Context" because it can actually be set by users in the admin screen. Instead use Firm::getContextLabel()

Reply:
Firm model has contextLabel() instead getContextLabel() function, I use that.

Reply:
Ah, my apologies